src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2909,6 +2909,67 @@ curl -X GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Get Product Limits for Plan
+
+To keep services stable for all clients, LiveChat limits common resources. The limits are different for each pricing plan.
+Method `check_products_limits_for_plan` shows what resources exceed its limits for any given plan. This information is useful before downgrading the plan.
+
+Limited resources in LiveChat:
```

Comment:
the `groups_per_agent` and `group_chooser_groups` are oddly specific compared to the other ones on the list 😆 are we talking about the limits from [this doc](https://docs.google.com/spreadsheets/d/1Ph2XVc8GBkfZX44vhGS11zxqn-55yC_jEWw-h_XgLuw/edit#gid=0)?  

If yes, I'm wondering why exactly would e.g. the `queue for cont. chat` be included as the limit is the same for all licenses. Same for `active agents`?
Also there were discussions about the "banned customers" limit, I'm not sure if us talking openly that there's such limit wouldn't cause some fuss among our customers. Based on the doc it's 100k - would we want to return that in the response, and include this in the table?